spec/FilesRouter.spec.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+const path = require('path');
+
+describe('FilesRouter', () => {
+  describe('File Uploads', () => {
+    const V8_STRING_LIMIT_BYTES = 536_870_912;
+
+    let server;
+
+    beforeAll(async () => {
+      server = await reconfigureServer({
+        maxUploadSize: '1GB',
+        port: 8384,
+      });
+    });
+
+    afterAll(async () => {
+      // clean up the server for resuse
+      if (server && server.close) {
```

Comment:
Wouldn't it be enough to just call `await reconfigureServer()` to init the server with the default config?

Reply:
Your right. Missed that, new to testing here.